src/main/java/org/broadinstitute/hellbender/tools/exome/germlinehmm/IntegerCopyNumberTransitionProbabilityCacheCollection.java

```diff
@@ -0,0 +1,180 @@
+package org.broadinstitute.hellbender.tools.exome.germlinehmm;
+
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.param.ParamUtils;
+
+import javax.annotation.Nonnull;
+import java.io.File;
+import java.io.Reader;
+import java.io.Serializable;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * This class constructs a collection of integer copy number transition probability caches, allowing
+ * streamlined query of the transition matrix for each contig, sex genotype, and distance.
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+public final class IntegerCopyNumberTransitionProbabilityCacheCollection implements Serializable {
```

Comment:
I agree but let's delegate it to later when we add the somatic HMM so that we can see the bigger picture.